test/basics.jl

```diff
@@ -153,3 +153,10 @@ end
     @test gs3 == [random(GRS, G) for _=1:30]
 
 end
+
+@testset "printing" begin
+    io = IOBuffer()
+    io = AbstractAlgebra.pretty(io)
+    print(io, AbstractAlgebra.Lowercase(), GapObj([1, 2, 3]))
```

Comment:
```suggestion
    io = GAP.AbstractAlgebra.pretty(io)
    print(io, GAP.AbstractAlgebra.Lowercase(), GapObj([1, 2, 3]))
```
AA is not a test dependency of GAP